hudi-hadoop-common/src/main/java/org/apache/hudi/common/util/OrcUtils.java

```diff
@@ -135,14 +113,37 @@ public List<Pair<HoodieKey, Long>> fetchRecordKeysWithPositions(HoodieStorage st
     return hoodieKeysAndPositions;
   }
 
-  @Override
-  public List<Pair<HoodieKey, Long>> fetchRecordKeysWithPositions(HoodieStorage storage, StoragePath filePath, Option<BaseKeyGenerator> keyGeneratorOpt) {
```

Comment:
why is this removed? Do we support record position encoding in ORC now?

Reply:
It is not removed, it is at Line 97 now since the main logic of this function is from the function in Line 92.